backend/geonature/core/imports/routes/__init__.py

```diff
@@ -12,3 +14,15 @@ def list_destinations():
     destinations = Destination.query.all()
     # FIXME: filter with C permissions?
     return schema.dump(destinations, many=True)
+
+
+@blueprint.route("/destination/<destinationCode>", methods=["GET"])
+@login_required
+def get_destination(destinationCode):
+    schema = DestinationSchema(only=["module"])
+    destination = db.session.execute(
```

Comment:
Consider handling the case where the destination is not found in the database. This could be done by returning a 404 error or a specific message to the client. [important]